site/content/en/docs/Start/macos.md

```diff
@@ -16,10 +16,20 @@ weight: 2
 
 If the [Brew Package Manager](https://brew.sh/) is installed, use it to download and install minikube:
 
-```shell
+If `which minikube` fails after installing with brew, you may need to uninstall the cask and reinstall by running:
```

Comment:
Can you move these changes to after `brew install minikube` and remove the lower part entirely? So that the flow will look like:

```
If the [Brew Package Manager](https://brew.sh/) is installed, use it to download and install minikube:

brew install minikube

If `which minikube` fails after installing with brew, you may need to uninstall the cask and reinstall by running:

brew cask uninstall minikube
brew install minikube
```

Reply:
OK

Reply:
Please check again.